packages/haul-cli/src/commands/multiBundle.ts

```diff
@@ -0,0 +1,184 @@
+import { Arguments } from 'yargs';
+import webpack from 'webpack';
+import {
+  Runtime,
+  getProjectConfigPath,
+  getNormalizedProjectConfigBuilder,
+  NormalizedProjectConfig,
+} from '@haul-bundler/core';
+import * as messages from '../messages/multiBundleMessages';
+import SimpleProgressWebpackPlugin from 'simple-progress-webpack-plugin';
+
+export default function ramBundleCommand(runtime: Runtime) {
+  return {
+    command: 'multi-bundle',
+    describe: 'Create multiple bundles to be used in multi-bundle mode',
+    builder: {
+      dev: {
+        description:
+          'If false, warnings are disabled and the bundle is minified (default: true)',
+        default: true,
+        type: 'boolean',
+      },
+      platform: {
+        description: 'Either "ios" or "android" (default: "ios")',
+        type: 'string',
+      },
+      minify: {
+        description:
+          'Allows overriding whether bundle is minified. This defaults to false if dev is true, and true if dev is false. Disabling minification can be useful for speeding up production builds for testing purposes.',
+        type: 'boolean',
+      },
+      'assets-dest': {
+        description:
+          'Directory name where to store bundles and assets referenced in the bundle.',
+        type: 'string',
+      },
+      'sourcemap-output': {
+        description: 'File name where to store generated source map',
+        type: 'string',
+      },
+      config: {
+        description: 'Path to the CLI configuration file',
+        type: 'string',
+      },
+      progress: {
+        description:
+          'Display bundle compilation progress with different verbosity levels',
+        // Ensure that we don't trip Xcode's error detection. 'verbose' is the
+        // only level that doesn't make Xcode think that the bundle failed.
+        default: !process.stdin.isTTY ? 'verbose' : 'compact',
+        choices: ['none', 'minimal', 'compact', 'expanded', 'verbose'],
+      },
+    },
+    async handler(
+      argv: Arguments<{
+        config?: string;
+        dev: boolean;
+        minify?: boolean;
+        platform: string;
+        assetsDest?: string;
+        bundleOutput?: string;
+        sourcemapOutput?: string;
+        progress: string;
+      }>
+    ) {
+      try {
+        const {
+          config,
+          dev,
+          minify,
+          platform,
+          assetsDest,
+          sourcemapOutput,
+          progress,
+        } = argv;
+
+        // TODO: figure out a better way to read and transpile user files on-demand
+        require('@haul-bundler/core-legacy/build/babelRegister');
```

Comment:
possibly could be moved to `getNormalizedProjectConfigBuilder ` now? (in a separate PR)